plugins/outputs/influxdb_v2/http.go

```diff
@@ -135,6 +147,20 @@ func (c *httpClient) Init() error {
 		c.concurrent = 1
 	}
 	c.pool = pond.NewPool(int(c.concurrent))
+
+	// setup observability
+	tags := map[string]string{
+		"url": c.url.String(),
+	}
+
+	c.statBytesWritten = c.statistics.Register("write", "bytes_total", make(map[string]string))
+	c.statSuccessfulWrites = c.statistics.Register(selfstatMeasurement, "successful_writes_total", tags)
+	c.statFailedWrites = c.statistics.Register(selfstatMeasurement, "failed_writes_total", tags)
+	c.statTimeout = c.statistics.Register(selfstatMeasurement, "timeouts_total", tags)
+	c.statRetryableErrorCounters = c.statistics.Register(selfstatMeasurement, "retryable_errors_total", tags)
+	c.statNonRetryableErrorCounters = c.statistics.Register(selfstatMeasurement, "non_retryable_errors_total", tags)
+	c.statSuccessfulWriteDuration = c.statistics.RegisterTiming(selfstatMeasurement, "successful_write_duration_ms", tags)
+	c.statFailedWriteDuration = c.statistics.RegisterTiming(selfstatMeasurement, "failed_write_duration_ms", tags)
```

Comment:
Please also convert those to the `write` or alternatively a `write_details` measurement! Furthermore, please reuse existing field names and/or follow the same naming scheme
```suggestion
	c.statSuccessfulWrites = c.statistics.Register("write", "writes", tags) // I would count the total number of writes here
	c.statFailedWrites = c.statistics.Register("write", "errors", tags)
	c.statRetryableErrorCounters = c.statistics.Register("write", "errors_retryable", tags)
	c.statNonRetryableErrorCounters = c.statistics.Register("write", "errors_non_retryable", tags)
	c.statTimeout = c.statistics.Register("write", "request_timeouts", tags)
	c.statSuccessfulWriteDuration = c.statistics.RegisterTiming("write", "request_success_time_ms", tags)
	c.statFailedWriteDuration = c.statistics.RegisterTiming("write", "request_fail_time_ms", tags)
```

Reply:
Hmmm, I looked at how our current metrics are named and that's my suggestion. I have mixed feelings about the Prometheus convention: On the one hand side it _is_ a written down convention so it would be easy for new developers to be linked to and to follow. However, on the other hand side, we do have existing metrics and an existing naming scheme (even though not written down but rather grown) so mixing two schemes would make it hard for users to get used to... I suggest to follow the existing scheme, i.e. what I suggested above. What do you think?